### DISCUSSION TITLE
Theia icons

### DISCUSSION THREAD

**Discussion by Carlosgrr** [gitkraken/vscode-gitlens#2950] [Q&A] [answered]
I am not sure if this should be a discussion or an issue, but the sidepanel icons are not showing in eclipse theia, here is my screenshots from vscodium and eclipse theia:
|vscodium|eclipse theia|
|--------|-------------|
|![image](https://github.com/gitkraken/vscode-gitlens/assets/2813789/46572045-a9e2-4ed3-bab3-84365312d746)|![image](https://github.com/gitkraken/vscode-gitlens/assets/2813789/6f4ea9eb-c189-4c32-b237-0bee6809596b)|

As can be seen in the above images, in my extension list, right below the cmake icon and above the tabnine icon is where the gitlens icons should be displayed, but doesn't show.

Is there some config I am missing, or a file?

I downloaded the "VSIX" file both from https://open-vsx.org and from this github repository and neither worked.

I am not sure if my environment is the problem, but I am running arch linux, and eclipse theia is running on firefox.

**Comment by d13** [comment, marked as answer]
I'm not super familiar with Eclipse Theia, but I quite sure it has nothing to do with your local env. I tested on Mac (Intel) with Firefox and Chrome, including several versions of GitLens, and ran into the same issue. 

I also noticed its not loading our Webviews (Home, Welcome, Commit Graph, Commit Details, etc) either.

To be fair, they do put a disclaimer on their Welcome screen: 
> Eclipse Theia Blueprint is not a production-ready product. Therefore, it is also not a replacement for Visual Studio Code or any other IDE.

I'm sure they're still working through extension support and would encourage opening a bug with them: https://github.com/eclipse-theia/theia/issues

**Reply by Carlosgrr** [reply]
Ok, I will do that, thank you!
